Translations/pt_BR.po

```diff
@@ -4434,314 +4507,219 @@ msgid "Bone Gasher"
 
 #: Source/monstdat.cpp:53
-#, fuzzy
-#| msgid "Satyr Lord"
 msgctxt "monster"
 msgid "Satyr Lord"
-msgstr "do senhor"
+msgstr "Senhor Sátiro"
 
 #: Source/monstdat.cpp:54 Source/monstdat.cpp:62
-#, fuzzy
-#| msgid "Flesh Clan"
 msgctxt "monster"
 msgid "Flesh Clan"
-msgstr "Carne rasgada de almas"
+msgstr "Clã Carne Fresca"
 
 #: Source/monstdat.cpp:55 Source/monstdat.cpp:63
-#, fuzzy
-#| msgid "Stone Clan"
 msgctxt "monster"
 msgid "Stone Clan"
-msgstr "de pedra"
+msgstr "Clã de Pedra"
 
 #: Source/monstdat.cpp:56 Source/monstdat.cpp:64
-#, fuzzy
-#| msgid "Fire Clan"
 msgctxt "monster"
 msgid "Fire Clan"
-msgstr "Fogo "
+msgstr "Clã de Fogo"
 
 #: Source/monstdat.cpp:57 Source/monstdat.cpp:65
-#, fuzzy
-#| msgid "Night Clan"
 msgctxt "monster"
 msgid "Night Clan"
-msgstr "da noite"
+msgstr "Clã da Noite"
 
 #: Source/monstdat.cpp:58
-#, fuzzy
-#| msgid "Fiend"
 msgctxt "monster"
 msgid "Fiend"
-msgstr ""
-"Pela Luz, eu ouvi falar desse demônio vil. Houve muitos que carregavam as "
-"cicatrizes de sua ira em seus corpos quando os poucos sobreviventes do "
-"ataque liderado por Lázarus se arrastaram para fora da Catedral. Eu não sei "
-"o que ele usou para cortar suas vítimas, mas não pode ser algo deste mundo. "
-"Deixou feridas podres com doenças, e mesmo eu as achei quase impossíveis de "
-"tratar. Cuidado se planeja batalhar contra esse demônio..."
+msgstr "Fiend"
 
 #: Source/monstdat.cpp:59
-#, fuzzy
-#| msgid "Blink"
 msgctxt "monster"
 msgid "Blink"
 msgstr "Cintilador"
 
 #: Source/monstdat.cpp:60
-#, fuzzy
-#| msgid "Gloom"
 msgctxt "monster"
 msgid "Gloom"
-msgstr "Armadura da penumbra"
+msgstr "Melancolia"
 
 #: Source/monstdat.cpp:61
-#, fuzzy
-#| msgid "Familiar"
 msgctxt "monster"
 msgid "Familiar"
-msgstr ""
-"Isso parece familiar, de algum modo. Acredito lembrar de ler algo muito "
-"parecido com esse poema quando pesquisava a história das aflições "
-"demoníacas. Falava de um lugar de grande mal que... espera - você não está "
-"indo para lá, está?"
+msgstr "Familiar"
 
 #: Source/monstdat.cpp:66
-#, fuzzy
-#| msgid "Acid Beast"
 msgctxt "monster"
 msgid "Acid Beast"
 msgstr "Fera Ácida"
 
 #: Source/monstdat.cpp:67
-#, fuzzy
-#| msgid "Poison Spitter"
 msgctxt "monster"
 msgid "Poison Spitter"
 msgstr "Cospe-Veneno"
 
 #: Source/monstdat.cpp:68
-#, fuzzy
-#| msgid "Pit Beast"
 msgctxt "monster"
 msgid "Pit Beast"
-msgstr "do poço"
+msgstr "Fera do Poço"
 
 #: Source/monstdat.cpp:69
-#, fuzzy
-#| msgid "Lava Maw"
 msgctxt "monster"
 msgid "Lava Maw"
 msgstr "Gorja de Lava"
 
 #: Source/monstdat.cpp:70 Source/monstdat.cpp:474
-#, fuzzy
-#| msgid "Skeleton King"
 msgctxt "monster"
 msgid "Skeleton King"
-msgstr "Covil do Rei Esqueleto"
+msgstr "Rei Esqueleto"
 
 #: Source/monstdat.cpp:71 Source/monstdat.cpp:482
 msgctxt "monster"
 msgid "The Butcher"
 msgstr "O Açougueiro"
 
 #: Source/monstdat.cpp:72
-#, fuzzy
-#| msgid "Overlord"
 msgctxt "monster"
 msgid "Overlord"
-msgstr "Elmo do suserano"
+msgstr ""
```

Comment:
Is this a mistake? Missing translation?

Reply:
Actually, this is probably done by an automatic action, it borrowed the translation from "OverLord's Helm" and pasted it here.